wds-disable-pingbacks.php

```diff
@@ -35,11 +41,47 @@ function wds_disable_pingbacks( $methods ) {
  */
 function wds_disable_pingbacks_options( $screen ) {
 	if ( 'options-discussion' === $screen->id ) {
```

Comment:
Could rewrite the function to return early and then have the logic after the return.

```php
if ( 'options-discussion' !== $screen->id ) {
  return;
}
// Rest of your logic here...
```

Reply:
done